polygeist/tools/cgeist/Test/Verification/booleanmem.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: cgeist %s -w -O0 --function=* -S | FileCheck %s
+// XFAIL: *
+
+void lambda_capture(bool x, bool &y) {
+  const auto f = [x, &y]() { y = !x; };
+  f();
+}
```

Comment:
We simply fail here. The problem's we are creating a `ptr<memref<?xi8>>` for the second field and trying to store a `memref<?xi1>` fails. Creating an issue for this.

Reply:
https://github.com/intel/llvm/issues/8066 issue